### ISSUE TITLE
Populate past events

### ISSUE THREAD

**Issue by ericbhatti** [GDGKolachi/gdgkolachi.com#11]
Please fetch past events details from [here](https://www.facebook.com/pg/GDGKolachi/events/).

If an event's start date is in past we will show a photos button instead of registration link. Also for the events card, if the schedule array is empty, let's not show that link at all.

**Comment by TalhaKhatri** [comment]
@ericbhatti The events have been successfully populated. Please review.
@mashhoodr

**Comment by mashhoodr** [comment]
@TalhaKhatri wheres the PR? or is this just a firebase change?

**Comment by TalhaKhatri** [comment]
@mashhoodr Yes, this was just a firebase change. Just wanted feedback if the change had any mistakes or anything. Or if something needs to be changed.

**Comment by nawazishali** [comment]
@TalhaKhatri Run the app and you will see that event images are not the same size which in turn effects the height of event cards and makes them inconsistent. It's just a simple style fix you will need to do in order to make it look good. Ask here if you need any help with it. And there are other changes mentioned by @ericbhatti  which we need to implement as well.

**Comment by TalhaKhatri** [comment]
@nawazishali Sure! Let me look into it. I'll let you know if I need any assistance, thank you.
